sway/commands/permit.c

```diff
@@ -84,7 +84,7 @@ struct cmd_results *cmd_reject(int argc, char **argv) {
 		return error;
 	}
 
-	struct feature_policy *policy = get_policy(argv[0]);
+	struct feature_policy *policy = get_feature_policy(argv[0]);
```

Comment:
Yes, I missed that. 

Scenario: suppose a policy file indicates a reject target against a file which does not exist. if we ignore the resolve fail and use the verbatim policy target, and the target is then created as a symlink to elsewhere, the reject policy won't be found when get_pid_exe is used to look it up. 

How do you want to handle that?